### ISSUE TITLE
vsdbg default path not working

### ISSUE THREAD

**Issue by tmds** [Azure/vscode-kubernetes-tools#1002]
https://github.com/Azure/vscode-kubernetes-tools/pull/967 changed the default path of vsdbg to `~/vsdbg/vsdbg`.

This doesn't work. Expanding the `~` is the responsibility of the shell, and `kubectl exec` doesn't involve a shell.

For example:
```
$ /usr/bin/kubectl exec -i dotnet60-app-76bb77ff87-gvfpx -- echo '~/vsdbg/vsdbg'
~/vsdbg/vsdbg
```
It can be fixed by using a shell that causes the `~` to be expanded:
```
$ /usr/bin/kubectl exec -i dotnet60-app-76bb77ff87-gvfpx -- /bin/sh -c 'echo ~/vsdbg/vsdbg'
/opt/app-root/vsdbg/vsdbg
```

`vsdbg` should be started using:
```
kubectl exec -i <pod> -- /bin/sh -c '<path> --interpreter=vscode'
```

cc @itowlson @pantosha

**Comment by itowlson** [comment]
@pantosha Would you be able to take a look at this please?  I'm not very familiar with the .NET debugging component.  (If you don't have bandwidth let me know and I'll go with @tmds' fix.)

**Comment by pantosha** [comment]
I'll take a look during current week

**Comment by itowlson** [comment]
Fixed via #1004.
